contracts/contracts/strategies/CompoundStrategy.sol

```diff
@@ -12,23 +12,8 @@ import {
 } from "../utils/InitializableAbstractStrategy.sol";
 
 contract CompoundStrategy is InitializableAbstractStrategy {
-    event RewardTokenCollected(address recipient, uint256 amount);
     event SkippedWithdrawal(address asset, uint256 amount);
 
-    /**
-     * @dev Collect accumulated reward token (COMP) and send to Vault.
-     */
-    function collectRewardToken() external onlyVault {
```

Comment:
This get removed by accident?

Reply:
That I threw into the base class could pull it back up.

Reply:
Nah good idea 👍